server/engine/src/main/antlr4/org/eclipse/lsp/cobol/core/CobolPreprocessor.g4

```diff
@@ -11,22 +11,7 @@ options {tokenVocab = CobolPreprocessorLexer;}
 
 startRule
    : .*? ((includeStatement | copyStatement | replaceAreaStartOrOffStatement | titleDirective | enterDirective
-   | controlDirective | linkageSection | plusplusIncludeStatement | procedureDivision | workingStorageSection)+ .*?)* EOF
-   ;
-
-// procedure devision for resolving predefined labels
-procedureDivision
-   : PROCEDURE DIVISION .*? DOT_FS
-   ;
-
-// linkage section for resolving predefined variables
-linkageSection
-   : LINKAGE SECTION DOT_FS
-   ;
-
-// working storage section for resolving speacal registers as variables
-workingStorageSection
-   : WORKING_STORAGE SECTION DOT_FS
+   | controlDirective | plusplusIncludeStatement)+ .*?)* EOF
```

Comment:
I wonder if we could turn this into a simple `(includeStatement | copyStatement | ... | ~(COPY|PLUSPLUSINCLUDE|...))*` rule.
`EXEC SQL INCLUDE` would need to be turned into a single token (similarly to the cics case).